src/vast_tracker.js

```diff
@@ -314,9 +314,14 @@ export class VASTTracker extends EventEmitter {
    * If an [ERRORCODE] macro is included, it will be substitute with errorCode.
    *
    * @param {String} errorCode - Replaces [ERRORCODE] macro. [ERRORCODE] values are listed in the VAST specification.
+   * @param {Boolean} [isCustomError=false] - Flag to allow custom values on error code.
    */
-  errorWithCode(errorCode) {
-    this.trackURLs(this.ad.errorURLTemplates, { ERRORCODE: errorCode });
+  errorWithCode(errorCode, isCustomError = false) {
```

Comment:
in the doc you call the new param `isCustomCode ` while here `isCustomError `. 

Beside the discrepancy (which needs to be fixed), I'd go for `isCustomCode`